browser/src/GenePage/VariantCooccurrenceCountsTable.spec.tsx

```diff
@@ -7,13 +7,58 @@ import {
 } from '../__factories__/VariantCooccurrenceCountsPerSeverityAndAf'
 
 import VariantCooccurrenceCountsTable from './VariantCooccurrenceCountsTable'
+import { forDatasetsMatching, forDatasetsNotMatching } from '../../../tests/__helpers__/datasets'
+
+const v2Regexp = /_r2/
+
+forDatasetsNotMatching(
+  v2Regexp,
+  'VariantCoocurrenceCountsTable with non v2 dataset "%s"',
+  (datasetId) => {
+    test('has no unexpected changes and renders as placeholder text', () => {
+      const heterozygousCounts =
+        HeterozygousVariantCooccurrenceCountsPerSeverityAndAfFactory.build()
+      const homozygousCounts = HomozygousVariantCooccurrenceCountsPerSeverityAndAfFactory.build()
+      const tableContent = render(
+        <VariantCooccurrenceCountsTable
+          datasetId={datasetId}
+          heterozygous_variant_cooccurrence_counts={heterozygousCounts}
+          homozygous_variant_cooccurrence_counts={homozygousCounts}
+        />
+      )
+      const normalContentFragment = tableContent.asFragment()
+      expect(normalContentFragment.querySelectorAll('p').length).toEqual(1)
+      expect(normalContentFragment.querySelectorAll('table').length).toEqual(0)
```

Comment:
I like this combo of DOM queries and a snapshot, I had always thought of them as mutually exclusive but thinking about it they have complementary strengths.